experiments/pages/lightningModalLauncher/lightningModalLauncher.xml

```diff
@@ -0,0 +1,202 @@
+<skuid__page unsavedchangeswarning="yes" personalizationmode="server" showsidebar="true" showheader="true" theme="Lightning">
+	<models>
+		<model id="Contact" limit="1" query="true" createrowifnonefound="false" datasource="salesforce" sobject="Contact">
+			<fields>
+				<field id="Id"/>
+				<field id="Name"/>
+			</fields>
+			<conditions/>
+			<actions/>
+		</model>
+		<model id="FlowParams" limit="20" query="true" datasource="Ui-Only" createrowifnonefound="true">
+			<fields>
+				<field id="name" displaytype="TEXT" length="255"/>
+				<field id="type" displaytype="TEXT" length="255"/>
+				<field id="value" displaytype="TEXT" length="255"/>
+			</fields>
+			<conditions/>
+			<actions/>
+		</model>
+		<model id="Flow" limit="20" query="true" datasource="salesforce" sobject="FlowInterview">
+			<fields>
+				<field id="Name"/>
+			</fields>
+			<conditions>
+				<condition type="fieldvalue" value="" enclosevalueinquotes="true" field="Name" operator="=" clientorserver="server" state="filterableon" inactive="false" name="Name"/>
+			</conditions>
+			<actions/>
+		</model>
+	</models>
+	<components>
+		<skuid__buttonSet uniqueid="sk-1c1D-18257" position="center" model="Contact">
+			<groups>
+				<skuid__buttonGroup displayType="detached">
+					<buttons>
+						<skuid__button label="Launch Lightning Modal" uniqueId="civilianTrainingButton">
```

Comment:
I suggest making this more generic:

```suggestion
						<skuid__button label="Launch Lightning Modal" uniqueId="launchLightningModalButton">
```

Reply:
user error on my end. the final file didn't save the last fixes with the corrected ID: `launchLightningModalButton`